### DISCUSSION TITLE
Very Slow GMail/IMAP login/downloading

### DISCUSSION THREAD

**Discussion by BobDenny** [kewisch/gdata-provider#368] [Support]
I know this seems strange. For a while I have been  seeing my T-Bird having trouble logging into my Google Apps for Work GMail account, especially when accessing Drafts, and saving a draft. But just opening a folder, Inbox even, it just gets stuck with the little bouncing ball in the tab going back and forth forever. Your Provider is set to the same Google Apps for Work Calendar. I was going crazy, so I decided  to disable all of my Extensions (and I only have three). By process of elimination I determined that the Provider for Google Calendar is causing this. I have enabled it, tested, disabled, tested, and back and forth 4 times, just to be sure I am not dreaming. I am using Provider 91.0.2 on Thunderbird 91.5.0 (32-bit) Windows 7 and this system is squeaky clean, never gives trouble.  With the provider disabled, and my others (tbSync and ThunderHTMLEdit) enabled Thunderbird works great. I wish I could be more specific.

**Comment by wsmwk** [comment]
@BobDenny Are you still seeing this problem when using a newer version of Thunderbird and a newer version of provider?

**Comment by wsmwk** [comment]
> For a while I have been seeing my T-Bird having trouble logging into my Google Apps for Work GMail account, especially when accessing Drafts, and saving a draft. But just opening a folder, Inbox even, it just gets stuck with the little bouncing ball in the tab going back and forth forever.

We'd need an update regarding your status using the latest version 102.
